SSA/Projects/InstCombine/Base.lean

```diff
@@ -246,19 +246,22 @@ def MOp.outTy : MOp φ → MTy φ
 instance : OpSignature (MOp φ) (MTy φ) where
   signature op := ⟨op.sig, [], op.outTy⟩
 
+
 @[simp]
 def Op.denote (o : Op) (arg : HVector TyDenote.toType (OpSignature.sig o)) :
     (TyDenote.toType <| OpSignature.outTy o) :=
   match o with
   | Op.const _ val => const? val
   | Op.and _ => pairBind and? arg.toPair
   | Op.or _ => pairBind or? arg.toPair
-  | Op.xor _ => pairBind xor? arg.toPair
+  | Op.xor _ => let p := arg.toPair
```

Comment:
why this change? isn't it better to keep the consistency with the other functions?

Reply:
They are now all consistent.